### ISSUE TITLE
Custom request timeout for TF-Serving

### ISSUE THREAD

**Issue by VladimirEI** [aws/sagemaker-tensorflow-training-toolkit#46]
**Situation**: I run heavy TF model, which require long inference times.
**Problem**: For each inference request, I get:
```
ERROR in serving: ExpirationError(code=StatusCode.DEADLINE_EXCEEDED, details="Deadline Exceeded")
```

`request_timeout` is by default set to 10 secongs, according to [proxy_client.py](https://github.com/aws/sagemaker-tensorflow-containers/blob/master/src/tf_container/proxy_client.py#L34):
```python
class GRPCProxyClient(object):
    def __init__(self, tf_serving_port, host='localhost', request_timeout=10.0, model_name=GENERIC_MODEL_NAME,
                 input_tensor_name=PREDICT_INPUTS, signature_name=DEFAULT_SERVING_SIGNATURE_DEF_KEY):
```

Transformer for TF is created with only port specified, so timeout is default, 10 seconds, according to [serve.py](https://github.com/aws/sagemaker-tensorflow-containers/blob/master/src/tf_container/serve.py#L97):
```python
def transformer(user_module):
    grpc_proxy_client = proxy_client.GRPCProxyClient(TF_SERVING_PORT)
    _wait_model_to_load(grpc_proxy_client, TF_SERVING_MAXIMUM_LOAD_MODEL_TIME_IN_SECONDS)

    return Transformer.from_module(user_module, grpc_proxy_client)
```

**Question**: Would it be possible to set custom timeout for TF-Serving?
(For now, I can do it by manually changing the code, but it would be nice to have it done properly)

**Comment by laurenyu** [comment]
hi @VladimirEI, thanks for using SageMaker! Definitely sounds like a useful thing to have, and we're always reprioritizing our backlog based on customer requests.

**Comment by prachiAeromana** [comment]
Any progress on this? I'm running into the same issue.

**Comment by zjost** [comment]
Same problem for batch transform.

**Comment by monark12** [comment]
Yeah, I am facing a similar issue. Getting a DeadLine Exceeded Error when running transform jobs on a large file.

**Comment by ChoiByungWook** [comment]
https://github.com/aws/sagemaker-tensorflow-container/pull/135

This should allow for individuals to customize their timeout using an environment variable.

**Comment by laurenyu** [comment]
looks like this was addressed in #135.
